docs/guides/custom-images.rst

```diff
@@ -0,0 +1,68 @@
+Using Custom Docker Images on Compute Engine
+=====================
+
+When running workflows remotely on Compute Engine, your tasks run in a container that uses one of the
+``orquestra-sdk-base`` images depending on whether you have requested a GPU or not (you can see their source code
+`here <https://github.com/zapatacomputing/orquestra-workflow-sdk/blob/main/docker/Dockerfile>`_ and
+`here <https://github.com/zapatacomputing/orquestra-workflow-sdk/blob/main/docker/cuda.Dockerfile>`_).
+If your task needs extra dependencies that you cannot install using imports (such as native libraries or
+executable binaries), you can use one of the above images as your base image to publish a new image that you
+will use as your container when running your task.
+
+Building an Image
+-----------------
+
+The base images are hosted at Orquestra Nexus. There are two versions of the base image per each SDK version, one with
+CUDA dependencies installed and one without. Their names follow the pattern
+``hub.nexus.orquestra.io/zapatacomputing/orquestra-sdk-base:<SDK version>[-cuda]``. For example,
+``hub.nexus.orquestra.io/zapatacomputing/orquestra-sdk-base:0.48.0`` or
+``hub.nexus.orquestra.io/zapatacomputing/orquestra-sdk-base:0.49.0-cuda``. You should pick the one that matches
+your SDK version and your GPU needs (``-cuda`` needs to be picked for GPU workflows).
+
+The base images run as a user named ``orquestra`` with uid ``1000``. Since these are Ubuntu based images, you
+can install any package that you need from Ubuntu repositories (by doing ``RUN apt install <package name>``).
+However, before doing so, you need to temporarily switch to the root user (via ``USER root`` directive) and
+restore back to user ``orquestra`` once you're done (``USER orquestra``).
+
+.. note::
+
+    You can use `standard OCI annotations <https://github.com/opencontainers/image-spec/blob/main/annotations.md>`_ to add metadata to your images.
+
+
+In order for the Compute Engine to run your workflow code, you should not have a ``ENTRYPOINT`` or ``CMD`` statement in
+your images. In case you do, these will be overriden at run time and you might observe unexpected behavior.
+
+Publishing the Image
+--------------------
+
+Once you have your ``Dockerfile`` ready, you can publish your image to our Nexus repository by using standard Docker tools.
+Please refer to `this page <https://zapatacomputing.atlassian.net/wiki/spaces/~61209e4528ae75006af8a1b8/pages/619577422/Nexus+Starts+Here>`_
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought:** ‏Copying what I said on Slack that this link isn't externally accessible.

Reply:
I think that should be OK for the moment as we have internal users only. @wdapalmazapata was thinking about moving it to a more general space in Confluence. William, do you think we should merge that page's content here or maybe make that page public in Confluence (not sure if that's possible)?

Reply:
I will be moving the "Nexus Stats Here" to a non-personal space in Confluence. However, as you indicated, it won't be a public page, and the same can be said about `hub.nexus.orquestra.io` at this point. For now, let's keep these instructions to be consumed by internal users only. In the future, we should look into making these Nexus pages consumable via Portal-hosted docs so Orquestra users in general can get to them.